packages/communicator-email-ses/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@dbos-inc/communicator-email-ses",
+  "version": "0.0.0-placeholder",
+  "description": "Communicator library - email in AWS with SES",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dbos-inc/dbos-transact",
+    "directory": "packages/communicator-email-ses"
+  },
+  "homepage": "https://docs.dbos.dev/",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "tsc --project tsconfig.json",
+    "test": "npm run build && jest --detectOpenHandles",
+    "setversion": "grunt setversion"
+  },
+  "dependencies": {
+    "@aws-sdk/client-ses": "^3.533.0"
```

Comment:
I think we should be using the v2 https://github.com/aws/aws-sdk-js-v3/tree/main/clients/client-sesv2 ?

Reply:
SESv2 has 1/3 the popularity and is 20% more bloaty, but I guess we _should_.